bin/build-sprite.js

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs';
+import path from 'path';
+import icons from '../dist/icons.json';
+import buildSprite from './build-sprite-function';
```

Comment:
I'm not in love with calling this file `build-sprite-function`. I can't think of a better title right now though. I'll have to think about this one.

Reply:
Maybe `buildSpriteString` and `build-sprite-string.js` since this function returns a string? I think I like that better than `build-sprite-function.js`.